lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -117,9 +117,62 @@ module.exports = [
         'font-size': {
           score: 1,
           details: {
-            items: {
-              length: 6,
-            },
+            items: [
+              {
+                source: 'http://localhost:10200/seo/seo-tester.html?extra_header=link%3D%253Chttp%253A%252F%252Flocalhost%253A10200%252Fseo%252F%253E%253B%2Brel%253D%2522canonical%2522:24:12',
+                selector: '.small',
+                coverage: '1.32%',
```

Comment:
I fear the person who has to update a character on this page next 😬 I guess it's not a number so we can't use our `<`? how important do we think this is?

Reply:
I'll remove